tests/mqtt-limit-msg-fail/suricata.yaml

```diff
@@ -0,0 +1,16 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular
+      filename: eve.json
+      types:
+        - mqtt:
+            msg-log-limit: XXX
```

Comment:
I am not sure this deserves a SV test, but fait enough for me

Reply:
This is to test if Suricata actually exits at startup with an error if this is not a  valid value. I think this is better done here than in an unit test.